react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,21 @@
 import { isArray } from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
+  if (args.length) {
+    if (args.length > 1) return Math.min(...args);
+    if (isArray(args[0])) return Math.min(...args[0]);
+    return Math.min(args[0]);
+  }
+  return undefined;
 }
 
-export function copy() {
+export const copy = param => Object.assign(param.constructor(), param);
 
+export const reverseMerge = (arr1, arr2) => [...arr2, ...arr1];
+
+export function filterAttribs(arg) {
+  const copyArg = copy(arg);
+  delete copyArg.a;
+  delete copyArg.b;
```

Comment:
This can and should be achieved by using a combination of spread operator and destructuring.
Can you came up with a solution?